src/Detector/DotNetCore/DotnetCoreDetector.cs

```diff
@@ -94,7 +94,7 @@ internal static string DetermineRuntimeVersion(string targetFramework)
             // Support .NET moniker aliases
             // Ex: "472" => "4.72"
             //     "48" => "4.8"
-            if (targetFramework.IndexOf('.') == -1)
+            if (targetFramework.Contains('.') == false)
```

Comment:
nit: any reason for this change? If so, we should consider removing the `== false`:

```cs
if (!targetFramework('.'))
```